plaidml/core/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ pml_cc_library(
     pmlc::ast
     pmlc::compiler
     pmlc::util
+    pmlc::target::all
```

Comment:
We can't do this. We need to allow for external projects to decide which targets they want to include.